### ISSUE TITLE
Support for custom/additional paths

### ISSUE THREAD

**Issue by Seldaek** [cerebroapp/cerebro-basic-apps#4]
It would be great to be able to add to the list of paths scanned by this. On windows I have some "portable" apps (i.e. not installed ones in Program Files, but just some apps dropped in a random folder). If I could add the couple folders to DIRECTORIES so that they get scanned for exe/lnk as well that would make Cerebro way more useful. Not too sure how to go about adding config options though at the moment so I am reporting this as an issue.

**Comment by PaulBurridge** [comment]
I agree this is currently stopping me using Cerebro. Most other launchers allow configuration of scanned directories, we need the ability to both add new directories and disable default directories.

For my personal workflow I place links to all applications I'd like to quick launch in a single directory and then point the launcher at that directory, disabling the default directories.

**Comment by KELiON** [comment]
Sounds great, I think shouldn't be a problem to make directories configurable using cerebro plugin settings

**Comment by alabotski** [comment]
Is it realese?

**Comment by ogustavo-pereira** [comment]
This suggestion is interesting, today is not a priority for our team. If you're interested in raising any PR, I'll keep it open until next month.

**Comment by c1phercraft** [comment]
Same as @PaulBurridge-Nasstar, the lack of being able to add paths to directories with executables, makes Cerebro less useful to me. On Linux I have a few, like `~/bin` where I store appimages and certain scripts. Cerebro currently does not find them, even though the are in my `PATH`.

**Comment by Jammmmm** [comment]
Putting in my request for this feature.

I currently use Launchy, which allows me to configure the exact paths that I want to be scanned. It works great, but is no longer maintained. Cerebro seems like a good replacement but I'd need to be able to define the required paths.
